multicluster/config/samples/clusterset_init/multicluster_clusterset_template.yaml

```diff
@@ -20,9 +20,4 @@ metadata:
 spec:
   leaders:
     - clusterID: test-cluster-leader
-  members:
-    - clusterID: test-cluster-member
-      serviceAccount: antrea-mc-member-access-sa
-    - clusterID: test-cluster-leader
-      serviceAccount: antrea-mc-member-access-sa
   namespace: antrea-multicluster
```

Comment:
@luolanzone: Question - do we still need `namespace` in leader ClusterSet?

BTW, I feel it is easier to understand if we move `namespace` to `leaders`, or at least name it `leaderNamespace`.

Reply:
Leader cluster set controller has a step to validate the Namespace, I feel it's indeed unnecessary for leader. But it's required by member, since both leader and member use the same definition, we may still keep it. 
We can rename it, but I think it requires ClusterSet version upgrade.

Reply:
Ok. We can consider change it together when we need to bump the API version later.